lib/Twig/Extension/Templating/PimcoreUrl.php

```diff
@@ -92,14 +92,13 @@ protected function generateUrl(array|string $name = null, ?array $parameters = [
         /** @var Concrete $object */
```

Comment:
```suggestion
        /** @var Concrete | null $object */
```
This should fix the Stan error.